src/linear_algebra/bilinear_form.lean

```diff
@@ -9,6 +9,7 @@ import linear_algebra.matrix.basis
 import linear_algebra.matrix.nondegenerate
 import linear_algebra.matrix.nonsingular_inverse
 import linear_algebra.tensor_product
+import linear_algebra.matrix.to_linear_equiv
```

Comment:
To prevent conflicts with #10349 (and because sorting imports is my hobby):
```suggestion
import linear_algebra.matrix.to_linear_equiv
import linear_algebra.tensor_product
```